### DISCUSSION TITLE
Feature Request: data saver mode

### DISCUSSION THREAD

**Discussion by benthillerkus** [community/community#6609] [Mobile] [answered]
Many repos are using GIFs in their `README.md` files. Unfortunately those are super heavy in file size and just today, by looking at some recommended repos for like 10 minutes, GitHub Mobile consumed 42 mB.

So I'd love to see an option that has the app download images over 50 kB only when the user clicks on them.

**Comment by aconsuegra** [comment, marked as answer]
Hi @benthillerkus
sorry for the late reply, and thanks for getting in touch with your feedback. I really like the idea and I think it makes a lot of sense! I'll create an issue to track this request in our end.

**Comment by alenpaulvarghese** [comment]
> Many repos are using GIFs in their `README.md` files. Unfortunately those are super heavy in file size and just today, by looking at some recommended repos for like 10 minutes, GitHub Mobile consumed 42 mB.
> 
> So I'd love to see an option that has the app download images over 50 kB only when the user clicks on them.

same case for me, 88MB gone within 15min of usage.
